backend/src/main/java/com/redhat/cloud/notifications/config/BackendConfig.java

```diff
@@ -42,7 +42,7 @@ private static String toggleName(String feature) {
     @Deprecated(forRemoval = true, since = "To be removed when we're done migrating to Unleash in all environments")
     boolean unleashEnabled;
 
-    @ConfigProperty(name = "notifications.enforce-bg-name-unicity", defaultValue = "false")
+    @ConfigProperty(name = "notifications.enforce-bg-name-unicity", defaultValue = "true")
```

Comment:
Why changing the default value? we will enable it through unleash then delete the flag no?